### ISSUE TITLE
[OOB 5.6] Program hangs by pressing USER button on DISCO_F429ZI

### ISSUE THREAD

**Issue by toyowata** [ARMmbed/mbed-os-example-uvisor#55]
### Description
- Type: Bug
- Related issue: none
- Priority: Major

---------------------------------------------------------------
## Bug

**Target**
DISCO_F429ZI

**Toolchain:**
GCC_ARM

**Toolchain version:**
arm-none-eabi-gcc.exe (GNU Tools for ARM Embedded Processors 6-2017-q1-update) 6.3.1 20170215 (release) [ARM/embedded-6-branch revision 245512]

**mbed-cli version:** 
1.2.0

**mbed-os sha:**
$ mbed ls
mbed-os-example-uvisor (079c01576664)
`- mbed-os (a4056fb8bf3e)

**DAPLink version:**
Version: 0221
Build:   Nov 19 2015 15:23:07

**Expected behavior**

Secure LED toggles by pressing USER button.

**Actual behavior**

When pressing USER button, program hangs most of time.  See log file below:

```
***** IRQ blinky uVisor example *****
Main loop count: 0
Main loop count: 1
Main loop count: 2
Main loop count: 3

Pressed switch, printing from interrupt - LED now on.
```

Note that I did not see any problem with master branch, only in mbed-os-5.6-oob branch.  Also, K64F does not have such an issue in mbed-os-5.6-oob branch.

**Steps to reproduce**

$ mbed import mbed-os-example-uvisor
$ cd mbed-os-example-uvisor
$ mbed update mbed-os-5.6-oob
$ mbed compile -m DISCO_F429ZI -t GCC_ARM -f

**Comment by ciarmcom** [comment]
ARM Internal Ref: IOTSEC-443

**Comment by Patater** [comment]
This is caused by https://github.com/ARMmbed/uvisor/issues/420

The version of Mbed OS being tested with attempts to sleep from unprivileged mode, but that causes an access violation.

**Comment by toyowata** [comment]
Fixed in the mbed-os-5.6-oob2.
